languages.adoc

```diff
@@ -20,76 +20,86 @@ SPDX-License-Identifier: Apache-2.0
 
 == Overview 
 
-uProtocol Language specific libraries (_libraries_) are written for a given programming language (ex. Java, C++, etc...) to implement the protocol and are the primary library that is used by applications and services (uEs) to communicate with other uEs. The _libraries_ declare the API for the uProtocol Transport Layer (uP-L1) and implement the uProtocol Communication Layer API (uP-L2).
+uProtocol Language specific libraries (_libraries_) are written for a given programming language (ex. Java, C++, etc...) to implement the protocol and are the primary library that is used by applications and services (uEs) to communicate with other uEs. The _libraries_ declare the API for the uProtocol Transport (uP-L1) and Communication Layers (uP-L2) and provide a generic implementation of the latter.
 
 
 == Requirements
 
 === Content
 
-[.specitem,oft-sid="req~up-language-api~1",oft-needs="impl,utest"]
+A language library
+
+[.specitem,oft-sid="req~up-language-transport-api~1",oft-needs="impl",oft-title="Library declares Transport Layer API"]
 --
-* *MUST* implement serializers, validators, and builders for the link:../basics/README.adoc[uProtocol data types] using the link:../up-core-api/README.adoc[up-core-api data model].
+* *MUST* declare the language specific link:up-l1/README.adoc[uProtocol Transport Layer interface], such that the interface can be implemented by the transport libraries.
 --
 
-[.specitem,oft-sid="req~up-language-transport-api~1",oft-needs="impl,utest"]
+[.specitem,oft-sid="req~up-language-comm-api~1",oft-needs="impl",oft-title="Library declares Communication Layer API"]
 --
-* *MUST* declare the language specific link:up-l1/README.adoc[uProtocol Transport Layer interface], such that the interface can be implemented by the transport libraries.
+* *MUST* declare the language specific link:up-l2/README.adoc[uProtocol Communication Layer (uP-L2) API], such that the interface can be implemented by the transport libraries.
```

Comment:
Transport libraries do not implement L2 APIs

Reply:
FMPOV they can indeed. Why shouldn't they? If there is a better, transport specific way to implement the RpcClient (see zenoh), then why not?

Reply:
@stevenhartley any further thoughts?

Reply:
```suggestion
* *MUST* declare the language specific link:up-l2/README.adoc[uProtocol Communication Layer (uP-L2) API], such that the interface *MAY* be implemented by the transport libraries.
```